sass/_FacetValues.scss

```diff
@@ -241,6 +242,7 @@
   right: 5px;
   background: white;
   @include defaultRoundedBorder();
+  border-color: grey;
```

Comment:
I think they are darker greys used elsewhere in the page. Like $color-dark-grey

Reply:
Yeah, at first I used `$color-dark-grey` and got even better contrast ratios, but I felt like `grey` was a less radical change. I could take some comparison screenshots of `$color-dark-grey` tomorrow.